src/Console/ConsoleErrorHandler.php

```diff
@@ -33,25 +33,19 @@ class ConsoleErrorHandler extends BaseErrorHandler
      */
     protected $_stderr;
 
-    /**
-     * Options for this instance.
-     *
-     * @var array
-     */
-    protected $_options;
-
     /**
      * Constructor
      *
      * @param array $options Options for the error handler.
      */
     public function __construct(array $options = [])
     {
-        if (empty($options['stderr'])) {
-            $options['stderr'] = new ConsoleOutput('php://stderr');
-        }
+        $options += [
+            'stderr' => new ConsoleOutput('php://stderr'),
```

Comment:
Is this lazy generated in php 7?

Reply:
IDK, what's your concern?

Reply:
Performance on lots of loops where this wouldnt be needed.
That was sure the reason this was lazy added via if statement.
Or is there some service singleton pattern on this, so this is only instant. once anyway?

Reply:
This is the console error handler for CLI, I doubt this will have any noticeable impact even over hundreds of calls.